lib/logwriter.c

```diff
@@ -1144,6 +1144,10 @@ log_writer_write_message(LogWriter *self, LogMessage *msg, LogPathOptions *path_
                                                     &consumed);
 
       self->partial_write = status == LPS_PARTIAL;
+      if (self->partial_write)
+       {
+         msg_debug("Partial message writer", evt_tag_printf("message", "%s", self->line_buffer->str));
```

Comment:
If it will be part of the final PR then I'd recommend to use `msg_trace` instead.

Reply:
I did not plan to keep it.